### PR TITLE
Reservations page

### PR DESCRIPTION
**Pull request by Uthmanbello**
## Highlights
- Render the reservations on the `My Reservations` page
- Style the table to display all necessary information
- Edit the background colors to have a uniform background color

### CODE REVIEW

File: src/components/my-reservations/myReservations.jsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  const combinedData = tutories.map((tutory) => {
+    const reservation = reservations.find((reservation) => reservation.classSubject_id === tutory.id);
+    return {
+      id: tutory.id,
+      subject: tutory.subject,
+      description: tutory.description,
+      date: reservation ? reservation.date : '',
+      city: reservation ? reservation.city : '',
+    };
+  });
```
Comment (enoqueJonas) [inline src/components/my-reservations/myReservations.jsx]: When I open the My Reservations page, It displays all classes if the one that were not reserved.
The page should only display reservations.

As you can see, I only have one reservation inserted in the database.
Screen Shot 2023-06-21 at 16 22 43

But it's displaying all the classes I have inserted.
Screen Shot 2023-06-21 at 16 23 07

Kindly fix the issue.

**Review by enoqueJonas — CHANGES_REQUESTED** [review]
Hi @Uthmanbello,
Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
 - No linter errors 
 - Gitflow was correctly used 
 - Good design
 
## Required Changes 
Check the comments under the review.

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: src/components/my-reservations/myReservations.jsx (hunk 1 above)
Comment (Uthmanbello) [reply src/components/my-reservations/myReservations.jsx]: @enoqueJonas , thanks a lot for the review, and sorry for the oversight. Will fix the issue ASAP

**Review by enoqueJonas — APPROVED** [review]
Hi @Uthmanbello, there's nothing else for me to say other than… Your project is complete, it's time to merge it🥳

# Status: Approved✅
68747470733a2f2f6d65646961322e67697068792e636f6d2f6d656469612f6c3366516333614356304871707a5763552f67697068792e676966

Your project is complete.

 ## Highlights
- Linters are passing
- You implemented the requested changes correctly

## Optional suggestions
Every comment with [optional] prefix won't stop the approval of the PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and happy coding👨🏾‍💻

*Feel free to leave any questions in the comments in the PR thread if something is not 100% clear. Please remember to tag me so I can receive the notification.*

**Pull request merged**